stock_vertical_lift/models/vertical_lift_operation_pick.py

```diff
@@ -83,15 +85,30 @@ def _domain_move_lines_to_do_all(self):
     def fetch_tray(self):
         self.current_move_line_id.fetch_vertical_lift_tray_source()
 
+    def get_next_move_line(self, order):
```

Comment:
```suggestion
    def _get_next_move_line(self, order):
```
As this method can't be used by the RPC layer (object returned can't be serialized).

Reply:
Done